docs/index.rst

```diff
@@ -0,0 +1,27 @@
+.. lstchain documentation master file, created by
+   sphinx-quickstart on Tue Aug 11 14:07:30 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+========================================================================
+ Repository for the high level analysis of the LST (:mod:`cta-lstchain`)
```

Comment:
Do we consider lstchain as high-level analysis?

Reply:
no, I changed it now